PhysicsTools/NanoAOD/python/autoNANO.py

```diff
@@ -35,5 +35,9 @@ def expandNanoMapping(seqList, mapping, key):
                   'customize': 'DPGAnalysis/MuonTools/muNtupleProducerBkg_cff.muDPGNanoBkgCustomize'},
     # PromptReco config: PHYS+L1
     'Prompt' : {'sequence': '@PHYS',
-                'customize': '@PHYS+@L1'}
+                'customize': '@PHYS+@L1'},
+    # Add lepton time-life info tables through customize combined with PHYS
+    'AddLepTimeLife' : {'customize': 'addTimeLifeInfo'},
```

Comment:
no need for `AddLepTimeLife` entry, one can do `NANO:@omething+@LepTimeLife`. 
please use `PhysicsTools/NanoAOD/leptonTimeLifeInfo_common_cff.addTimeLifeInfo` and remove the import at top level